Thu Jul 14 00:02:24 2022
Windows-10-10.0.19044
Ren'Py 8.0.0.22062402
 

Bootstrap to the start of init.init took 0.08s
Early init took 0.00s
Loader init took 0.06s
Loading error handling took 0.05s
Loading script took 0.20s
Loading save slot metadata. took 0.01s
Loading persistent took 0.00s
Faled to initialize steam: FileNotFoundError("Could not find module 'C:\\Users\\alext\\OneDrive - Universidade de Lisboa\\Documentos\\Programas\\renpy-8.0.0-sdk\\lib\\py3-windows-x86_64\\steam_api64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
Running init code took 0.13s
Loading analysis data took 0.02s
Analyze and compile ATL took 0.00s
Index archives took 0.00s
Dump and make backups. took 0.08s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc. took 0.03s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: Disabled thread optimizations.
Creating interface object took 0.45s
Cleaning stores took 0.00s
Init translation took 0.05s
Build styles took 0.00s
Load screen analysis took 0.05s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.10s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start took 0.16s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'NVIDIA Corporation'"
Renderer: b'NVIDIA GeForce RTX 3050 Laptop GPU/PCIe/SSE2'
Version: b'4.6.0 NVIDIA 512.78'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1739, 978) drawable=(1739, 978)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096
Cleaning stores took 0.00s
Init translation took 0.00s
Build styles took 0.01s
Load screen analysis took 0.03s
Analyze screens took 0.00s
Prepare screens took 0.00s
Running _start took 0.00s

Full traceback:
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\bootstrap.py", line 274, in bootstrap
    renpy.main.main()
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\main.py", line 663, in main
    renpy.persistent.update(True)
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\persistent.py", line 397, in update
    save()
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\persistent.py", line 413, in save
    renpy.loadsave.location.save_persistent(data)
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\savelocation.py", line 494, in save_persistent
    l.save_persistent(data)
  File "C:\Users\alext\OneDrive - Universidade de Lisboa\Documentos\Programas\renpy-8.0.0-sdk\renpy\savelocation.py", line 345, in save_persistent
    with open(fn_tmp, "wb") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\alext\\OneDrive - Universidade de Lisboa\\Documentos\\Projetos pessoais\\Gamedev\\SuNoFes_22\\game\\saves\\persistent.1657753343.tmp'

While running game code:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\alext\\OneDrive - Universidade de Lisboa\\Documentos\\Projetos pessoais\\Gamedev\\SuNoFes_22\\game\\saves\\persistent.1657753343.tmp'
